src/EnergyPlus/ZoneAirLoopEquipmentManager.cc

```diff
@@ -542,13 +484,13 @@ namespace ZoneAirLoopEquipmentManager {
                                     OutputProcessor::Unit::W,
                                     AirDistUnit(AirDistUnitNum).HeatRate,
                                     "System",
-                                    "Sum",
+                                    "Average",
```

Comment:
Fix #7595 - these have been wrong since, forever. This should be validated automatically, so posted #7596 .